### PR TITLE
fix(ui): CIDR blocks not working

### PR DESCRIPTION
**Pull request by specters312**
## Issue CIDR
I worked on this today to get CIDR to resolve I feel like since this is the first step in the process all of these different methods should work to not drive people away who want to use this tool.

### CODE REVIEW

**Comment by specters312** [comment]
This will resolve #123

File: web/api/views.py [hunk 1]
```diff
@@ -1112,40 +1113,59 @@
...
+            return Response({
+                'status': False,
+                'message': 'Invalid IP address or CIDR block'
+            })
+        except Exception as e:
+            logger.exception(e)
+            return Response({
+                'status': False,
+                'message': 'Exception {}'.format(e)
+            })
```
[the remark this answers is not in the document]
Comment (psyray) [reply web/api/views.py:1165]: Could you fix this one and not return a full exception message but a simple one, like 'An error as occurred, check the log file for more information'

**Comment by psyray** [comment]
Mmmh strange behavior.
I have scanned my 192.168.1.0/24 subnet and I've checked only hostname, not IPs, and all was imported.
Could you have a look at this also

image

**Comment by specters312** [comment]
> Mmmh strange behavior. I have scanned my 192.168.1.0/24 subnet and I've checked only hostname, not IPs, and all was imported. Could you have a look at this also
> 
> image

Yeah will look into this I think I was only expecting people to use external let me setup for internal as well :D

**Comment by psyray** [comment]
Could you fix this ASAP, we need to publish the 2.0.7 release in the next few days
Thanks

File: web/api/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnonymousWP) [reply web/api/views.py:1165]: Bump.

**Comment by AnonymousWP** [comment]
Maybe we should close this PR and fix it ourselves due to inactivity.

**Comment by psyray** [comment]
> Maybe we should close this PR and fix it ourselves due to inactivity.

Yes I'll do this ASAP

**Comment by psyray** [comment]
Closed due to inactivity, replaced by #141

**Pull request closed without merging**